products/models.py

```diff
@@ -0,0 +1,61 @@
+from django.db import models
+
+class Product(models.Model):
+    name_english          = models.CharField(max_length=100)
+    current_buying_price  = models.DecimalField(max_digits=18, decimal_places=2)
+    current_selling_price = models.DecimalField(max_digits=18, decimal_places=2)
+    author                = models.ForeignKey('Author', on_delete=models.SET_NULL, null=True)
+    size                  = models.ForeignKey('Size', on_delete=models.SET_NULL, null=True)
+    original_price        = models.DecimalField(max_digits=18, decimal_places=2)
+
+    class Meta:
+        db_table = 'products'
+
+class ProductTheme(models.Model):
+    theme   = models.ForeignKey('Theme', on_delete=models.SET_NULL, null=True)
+    product = models.ForeignKey('Product', on_delete=models.CASCADE)
+
+    class Meta:
+        db_table = 'product_themes'
```

Comment:
중간테이블은 `복수-복수` 형태로 만들어주세요~
```suggestion
class ProductTheme(models.Model):
    theme   = models.ForeignKey('Theme', on_delete=models.SET_NULL, null=True)
    product = models.ForeignKey('Product', on_delete=models.CASCADE)

    class Meta:
        db_table = 'products_themes'
```

Reply:
넵 수정했습니다!